composer.json

```diff
@@ -9,13 +9,13 @@
   ],
   "require": {
     "php": "^7.1|^8.0",
-    "goetas-webservices/soap-reader": "^0.3.1",
+    "goetas-webservices/soap-reader": "0.3.x-dev",
```

Comment:
I have tagged now `v0.3.3`, you can use that

Reply:
see https://github.com/goetas-webservices/soap-reader/releases/tag/v0.3.3

Reply:
done